stable/redis/templates/svc.yaml

```diff
@@ -24,5 +24,10 @@ spec:
   - name: redis
     port: 6379
     targetPort: redis
+  {{- if .Values.metrics.enabled }}
+  - name: metrics
+    port: 9121
+    targetPort: redis
```

Comment:
Is the target port really `redis`, i. e. 6379?

Reply:
You're right. I've updated it.